### PR TITLE
gnatmake, gnatprep, gprbuild: add page

### PR DESCRIPTION
**Pull request by dmbaturin**
Add pages for [Ada](https://ada-lang.io/) build tools. gnatmake and gnatprep are from the GNAT toolchain (which is also a part of the GCC compiler suit), while gprbuild is a separate tool developed by AdaCore and released under the GNU GPL.  

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
The page LGTM. I have made some suggestions for this PR.

File: pages/common/gnatmake.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Compile an executable:
+
+`gnatmake {{source_file1.adb source_file2.adb ...}}`
+
+- Set a custom executable name:
+
+`gnatmake -o {{executable_name}} {{source_file.adb}}`
+
+- Force recompilation:
```
[the remark this answers is not in the document]
Comment (dmbaturin) [reply pages/common/gnatmake.md:14]: `-F` is a valid flag, but it's meaning is different (it has to do with path formatting in error messages, but I have to admit I've never used it). Also, gnatmake doesn't support long flag names like `--force`, oddly enough.

File: pages/common/gnatprep.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# gnatprep
+
+> Preprocessor for Ada source code files (part of the GNAT toolchain).
+> More information: <https://gcc.gnu.org/onlinedocs/gnat_ugn/Preprocessing-with-gnatprep.html>.
+
+- Use symbol definitions from a file:
```
Comment (navarroaxel) [inline pages/common/gnatprep.md:6]: use for what?

File: pages/common/gprbuild.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# gprbuild
+
+> A high-level build tool for projects written in Ada and other languages (C/C++/Fortran).
+> More information: <https://docs.adacore.com/gprbuild-docs/html/gprbuild_ug.html>.
+
+- Build a project (assuming only one `*.gpr` file exists in the current directory):
+
+`gprbuild`
+
+- Specify the [P]roject file to build:
```
Comment (navarroaxel) [inline pages/common/gprbuild.md:10]:
```suggestion
- Build a specific [P]roject file:
```

File: pages/common/gnatprep.md (hunk 2 above)
Comment (dmbaturin) [reply pages/common/gnatprep.md:6]: For looking up values associated with symbol names and substituting those names in source code files.

I don't really understand how you want this wording improved. Are you implying it should be understandable for people who don't know what a preprocessor does, or your point is that it should be like "**Load** symbol definitions from a file"?

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**